### PR TITLE
Fetch api

### PR DESCRIPTION
**Pull request by sanika621879**
swagger document name : apidoc.yaml.
Please review.

### CODE REVIEW

File: apidoc.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ info:
   license:
     name: API License
     url: 'http://genkwiz.com'
-  version: 0.0.1
+  version: 1.0-oas3
```
Comment (rdsarda) [inline apidoc.yaml:12]: version should now be `0.0.2`.

File: apidoc.yaml [hunk 2]
```diff
@@ -57,11 +57,18 @@ paths:
                   id:
```
Comment (rdsarda) [inline apidoc.yaml:57]: Please add a description.

File: apidoc.yaml [hunk 3]
```diff
@@ -57,11 +57,18 @@ paths:
...
                     type: integer
                     format: int32
+                  aid:
```
Comment (rdsarda) [inline apidoc.yaml:60]: What does aid mean? What is it required for?

File: apidoc.yaml [hunk 4]
```diff
@@ -109,6 +116,39 @@ paths:
...
+        '403':
+          description: Forbidden
+          content: {}
+        '404':
+          description: Not Found
+          content: {}
+      deprecated: false
 components:
   schemas:
     Hint:
```
Comment (rdsarda) [inline apidoc.yaml:154]: the prefix hint is not required for the child properties under this object type.

File: apidoc.yaml [hunk 5]
```diff
@@ -191,6 +231,15 @@ components:
           format: float
         Type:
           type: string
+    Answer:
+      title: Answer of the question.
+      type: object
+      properties:
+        Id:
```
Comment (rdsarda) [inline apidoc.yaml:238]: Which id is this? Is this id needed on the UI?

File: apidoc.yaml [hunk 6]
```diff
@@ -55,13 +55,28 @@ paths:
...
                 quizId: '$response.body#/id'
+            GetAnswerByQuesId:
+              description: >-
+                The 'aid' value returned in the response can be used as the 'Id'
+                parameter in 'GET /answers/{Id}' to fetch anwser.
+              operationId: displayAnswer
+              parameters:
+                Id: '$response.body#/aid'
         '400':
           description: Invalid Order
```
Comment (vrushbru) [inline apidoc.yaml:81]: Change the description for error  code 400

File: apidoc.yaml [hunk 7]
```diff
@@ -55,13 +55,28 @@ paths:
                 type: object
                 properties:
                   id:
+                    description: quiz id .
+                    type: integer
+                    format: int32
+                  aid:
```
Comment (vrushbru) [inline apidoc.yaml:61]: Chang e the name aid

File: apidoc.yaml (hunk 5 above)
Comment (sanika621879) [reply apidoc.yaml:246]: Id: Question id
it is not needed on UI

Comment (rdsarda) [reply apidoc.yaml:246]: Okay. If its not needed on UI, please remove it.

Comment (sanika621879) [reply apidoc.yaml:246]: I have few doubts regarding this topic can we discuss about this in meeting??

File: apidoc.yaml (hunk 4 above)
Comment (sanika621879) [reply apidoc.yaml:162]: done

File: apidoc.yaml (hunk 3 above)
Comment (sanika621879) [reply apidoc.yaml]: I have updated the doc with descriptions pls review.

File: apidoc.yaml (hunk 6 above)
Comment (sanika621879) [reply apidoc.yaml]: done

File: apidoc.yaml (hunk 7 above)
Comment (sanika621879) [reply apidoc.yaml]: done

**Pull request closed without merging**